src/processes.h

```diff
@@ -28,9 +28,25 @@
 
 #include <sys/types.h> /* for pid_t */
 
+#define FORKFAILED -1
+#define NOINIT -2
+#define PROCSFULL -3
+#define NOCHILD -4
+
 typedef void (*process_func_t) (void *);
+
+void
+init_procs (int max);
```

Comment:
maybe it makes sense to apply a prefix to the methods defined here? Like procs_terminate_child, procs_init, ... to make it easier to find the implementation of those methods when looking for them.

Reply:
I did it for `terminate_childs` and `procs_init` as they currently are not used in the rest of openvas. Feel free to change their names too, when applying IPC